Resources/config/mailer.xml

```diff
@@ -10,7 +10,9 @@
     </parameters>
 
     <services>
-        <service id="fos_user.mailer.default" class="FOS\UserBundle\Mailer\Mailer" public="false">
+        <service id="fos_user.mailer.default" alias="fos_user.mailer.twig_swift" public="false" />
```

Comment:
this is a BC break, as it would change the implementation being used by existing projects.

and anyway, the one being used is defined in the Configuration class. We don't need an alias here to choose it.

Reply:
I was not sure where this parameter should be configured. Now it is clear: this will be a part of recipe. Thank you.

Reply:
@covex-nn It doesn't have to be part of the recipe, see https://github.com/FriendsOfSymfony/FOSUserBundle/pull/2723